projects/plugins/jetpack/extensions/shared/memberships/settings.js

```diff
@@ -138,6 +178,18 @@ export function NewsletterAccessRadioButtons( {
 							{ accessLabel }
 							{ reach }
 						</label>
+						{ /*
+						 * This adds a tier selector when:
+						 * - the paid_subscribers option is selected
+						 * - stripe is selected
+						 * - there are newsletter plans
```

Comment:
```suggestion
						 * - there are newsletter plans
```
More than 1 newsletter plan. Otherwise it works as it is working right now.